content/en/support/_index.md

```diff
@@ -7,6 +7,10 @@ type: docs
 
 Below is a list of companies that offer commercial products and services for Kyverno.
 
+## BlakYaks
+
+[BlakYaks](https://blakyaks.com) are a Microsoft Gold Partner speeding up enterprise container platform adoption, specialising in Azure, DevOps and cloud-native. We focus on building and managing enterprise-grade Azure, container, PaaS, and serverless platforms solutions that are secure, compliant and software-defined.
```

Comment:
Can you please add a line on what you offer for Kyverno (can be training, services, products, etc.)?